specs/altair/light-client/p2p-interface.md

```diff
@@ -59,7 +59,7 @@ New global topics are added to provide light clients with the latest updates.
 This topic is used to propagate the latest `LightClientFinalityUpdate` to light clients, allowing them to keep track of the latest `finalized_header`.
 
 The following validations MUST pass before forwarding the `finality_update` on the network.
-- _[IGNORE]_ No other `finality_update` with a lower or equal `finalized_header.slot` was already forwarded on the network
+- _[IGNORE]_ The `finalized_header.slot` is greater than the `slot` of all previously forwarded finality updates
```

Comment:
```suggestion
- _[IGNORE]_ The `finalized_header.slot` is greater than the one of all previously forwarded `finality_update`
```
- Use "the one" to make it clear that it is referring to the same `slot` (there is also `attested_header.slot`)
- Use `finality_update` for consistency ("finality update" is not used so far as a term in the spec, just the variable name)